tests/wpt/web-platform-tests/css/cssom-view/scroll-no-layout-box.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<meta charset=utf-8>
+<title>cssom-view - Scrolling element with no layout box</title>
+<link rel="help" href="https://drafts.csswg.org/cssom-view/#dom-element-scroll">
+<link rel="help" href="https://drafts.csswg.org/cssom-view/#css-layout-box">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+
+<div style="display: none">
```

Comment:
nit: `display: none;` -> `;`

Reply:
Why would that mater?

Reply:
No matter here. just mention that CSS usually with semicolon in the end.